Code Chefinne/Über Uns.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+</head>
+<div>
+    <img src="Images/Logo C.jpeg" alt="ChefinnenLogo" style=width:10%;height:100%;border-radius:50%;z-index:200/>
```

Comment:
A suggestion is to split the code to multiple lines (for src and alt) and to move the styling (e.g. width and height) to the CSS-file, for example using the class "image-logo"